just removed the HTML element and made clean

milanshresta opened the pull request:
[just removed the HTML element and made clean](https://github.com/up-for-grabs/up-for-grabs.net/commit/ec5f174d51c0e06628c6cf6b8dd5c9ae708d6dca)

milanshresta committed:
just removed the HTML element and made clean

milanshresta committed:
Merge branch 'up-for-grabs:gh-pages' into gh-pages

shiftkey commented on stylesheets/stylesheet.css:
```diff
@@ -41,87 +41,9 @@ MeyerWeb Reset
-video {
+*,
+::before,
+::after {
```
🤔 are there any downsides to including these psuedo-element selectors here?

milanshresta replied:
For better explanation: https://stackoverflow.com/questions/31317238/why-use-selector-in-combination-with-before-and-after

shiftkey reviewed (commented):
I haven't had a chance to look at this in a real browser, just curious about the benefits of tidying this up...

milanshresta committed:
added a pseudo elements with universal selector

Using a (*) universal selector doesn't affect the ::after and ::before pseudo elements. 

For a better explanation: https://stackoverflow.com/questions/31317238/why-use-selector-in-combination-with-before-and-after
